### PR TITLE
Unsubscribe on Instance termination

### PR DESCRIPTION
**Pull request by iamtanmay**
Added UUID Tag creation for Instances. CleanPVRole gives ec2:CreateTags permissions to Instances.

WIP Autoscaler script utils.py is extended to unsubscribe_node on Instance termination.

### CODE REVIEW

**Comment by iamtanmay** [comment]
Hey Jay,

Sorry for the messy pull requests before, its my first time.

I have made a fresh clone of develop branch and made my changes here. No conflicts this time :)

File: scripts/redhat_ose-register-3.10.sh [hunk 1]
```diff
@@ -23,5 +23,9 @@ subscription-manager repos --enable="rhel-7-server-rpms" \
     --enable="rhel-7-server-extras-rpms" \
     --enable="rhel-7-server-ose-3.10-rpms" \
     --enable="rhel-7-fast-datapath-rpms" \
-    --enable="rhel-7-server-ansible-2.4-rpms" \
-    --enable="rh-gluster-3-client-for-rhel-7-server-rpms"
+    --enable="rhel-7-server-ansible-2.4-rpms"
+
```
Comment (pierreozoux) [inline scripts/redhat_ose-register-3.10.sh]: You deleted this line too.

File: scripts/scaling/aws_openshift_quickstart/utils.py [hunk 2]
```diff
@@ -292,6 +292,7 @@ def process_pipeline(cls):
                 if category == 'combined':
                     continue
                 # cls.nodes_to_remove[category] is a list of instance IDs.
+		cls.unsubscribe_nodes(cls.nodes_to_remove[category], category)
```
Comment (pierreozoux) [inline scripts/scaling/aws_openshift_quickstart/utils.py]: Indentation.

**Comment by jaymccon** [comment]
Thanks for the contribution @iamtanmay, I'm going to merge so we can get CI run against this, I'll cc you on any issues that come up there.

**Pull request merged**